en/docs/learn/consume-api/manage-subscription/advanced-topics/update-subscription-tier.md

```diff
@@ -0,0 +1,23 @@
+# Update Subscription Tier
```

Comment:
This document should move to under manage subscriptions as this a main feature.

Reply:
Shall we rename this to Update API Subscription Tier

Reply:
resolved